app/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,9 @@
     <string name="clearButtonContentDescription">Clear search input</string>
     <string name="no_compatible_third_party_app_installed">No compatible app installed</string>
     <string name="addBookmarkMenuTitle">Add Bookmark</string>
+    <string name="addFavoriteMenuTitle">Add Favorite</string>
```

Comment:
did you mean to add it here and not to the untranslated strings file?

Reply:
yeah, the initial intention was to keep those strings within this file, merge the PR and once translations are ready merge them into code. But I see that it can be problematic, so moving back to untranslated strings everything here.